rotkehlchen/api/v1/types.py

```diff
@@ -11,3 +12,8 @@ class EvmTransactionDecodingApiData(TypedDict):
 class EvmPendingTransactionDecodingApiData(TypedDict):
     evm_chain: SUPPORTED_CHAIN_IDS
     addresses: Optional[list[ChecksumEvmAddress]]
+
+
+class IncludeExcludeFilterApiData(TypedDict):
+    values: list[HistoryBaseEntryType]
+    behaviour: Literal['include', 'exclude']
```

Comment:
Change this, so that after the API validation it becomes IN and NOT IN. This way it can go directly to the filter, without any extra ifs

Also not sure it works, just added a suggestion here. Can you have a default value so you don't need to add behaviour all the time?

(seems not: https://github.com/python/mypy/issues/6131)

```suggestion
    behaviour: Literal['IN'', 'NOT IN'] = 'IN'
```

In that case why not turn this into a dataclass isntead and intiialize it at marshmallow field validation?